pack/td.json

```diff
@@ -489,6 +617,80 @@
         "uniqueness": false
     },
     {
+        "code": "13035",
+        "cost": 2,
+        "deck_limit": 3,
+        "faction_code": "haas-bioroid",
+        "faction_cost": 3,
+        "flavor": "\"Strictly speaking, it isn't illegal to use an interrogation simulator as the blueprint for ICE.\" - Mason Bellamy",
+        "illustrator": "Andreas Zafiratos",
```

Comment:
**Andreas Zafiratos** is already referenced as an illustrator in nrdb. Better to correct FFG's mistake.